docs/custom-flows/oauth-connections.mdx

```diff
@@ -118,60 +83,94 @@ Clerk ensures that security critical nonces will be passed only to allowlisted U
-
-<Images
-  width={3024}
-  height={1654}
-  src="/docs/images/custom-flows/mobile-oauth-allowlist.jpg"
-  alt="The 'Social Connections' page of the Clerk Dashboard, with the 'Settings' tab selected. The 'Settings' tab shows a section titled 'allowlist for mobile OAuth redirect' and has an input for redirect URLs."
-/>
-
-## OAuth account transfer flow
-
-When a user initiates an OAuth verification during sign-in or sign-up, it may sometimes be necessary to move the verification between the two flows.
-
-For example, if someone already has an account, and tries to go through the sign up flow with the same OAuth account, they can't perform a successful sign-up again. Or, if someone attempts to sign in with their OAuth credentials but does not yet have an account, they won't be signed in to the account. For these scenarios, Clerk provides "account transfers."
-
-<CodeBlockTabs type="framework" options={["JavaScript"]}>
-```js
-// Get sign-in and sign-up objects in the OAuth callback page
-const { signIn, signUp } = window.Clerk.client;
-
-// If the user has an account in your application, but does not yet 
-// have an oauth account connected, you can use the transfer method to 
-// forward that information.
-
-const userExistsButNeedsToSignIn =
-  signUp.verifications.externalAccount.status === "transferable" &&
-  signUp.verifications.externalAccount.error?.code ===
-    "external_account_exists";
-
-if (userExistsButNeedsToSignIn) {
-  const res = await signIn.create({ transfer: true });
-  if (res.status === "complete") {
-    window.Clerk.setActive({
-      session: res.createdSessionId,
-    });
-  }
-}
-
-// If the user has an existing oauth account but does not yet exist as 
-// a user in your app, you can use the transfer method to forward that 
-// information.
-
-const userNeedsToBeCreated =
-  signIn.firstFactorVerification.status === "transferable";
-
-if (userNeedsToBeCreated) {
-  const res = await signUp.create({
-    transfer: true,
-  });
-  if (res.status === "complete") {
-    window.Clerk.setActive({
-      session: res.createdSessionId,
-    });
+To allowlist a redirect URL via the Clerk Dashboard:
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections).
+2. In the navigation sidebar, select **User & Authentication > Social Connections**.
+3. Select the **Setting** tab and add your redirect URL to the **Redirect URLs** list.
+
+## OAuth account transfer flows
+
+In some cases, a user may be trying to sign in with an OAuth account, but they don't have an account in your application yet. Or a user may be trying to sign up with an OAuth account, but they already have an account in your application. In these cases, Clerk provides "account transfers" which allow you to forward the user's information to the appropriate flow.
```

Comment:
```suggestion
In some cases, a user may try to sign in with OAuth without having an account with your app, or sign up with OAuth despite already having an account with your app. In these cases, Clerk provides "account transfers", which allow you to forward the user's information to the appropriate flow.
```
Simplified

Reply:
I can remove the quotes from account transfers, but I think the language I used is more clear.
The semantics you suggested are kind of confusing upon reading. I think it's better to break up the sentences and make the two use cases very clearly defined.

Reply:
Ignore my suggestion then